src/Umbraco.Web.Common/DependencyInjection/UmbracoBuilderExtensions.cs

```diff
@@ -108,8 +108,8 @@ public static partial class UmbracoBuilderExtensions
             var requestCache = new HttpContextRequestAppCache(httpContextAccessor);
             var appCaches = AppCaches.Create(requestCache);
 
-            services.ConfigureOptions<ConfigureKestrelServerOptions>();
             services.ConfigureOptions<ConfigureIISServerOptions>();
+            services.ConfigureOptions<ConfigureKestrelServerOptions>();
             services.ConfigureOptions<ConfigureFormOptions>();
```

Comment:
```suggestion
            services.ConfigureOptions<ConfigureKestrelServerOptions>();
            services.ConfigureOptions<ConfigureFormOptions>();
```

I don't think we actually need to run the configuration, since all that really gives us is a warning in the console 😄, from my testing at least, you can't inject it anywhere anyhow.